LedgerRecord is missing successful_transaction_count and failed_transaction_count

**Describe the bug**
The LedgerRecord interface is missing the two fields:
`successful_transaction_count`
`failed_transaction_count`

these two fields are listed in our [docs](https://developers.stellar.org/api/resources/ledgers/object/)  but missing from the js-sdk. They are however in the [go-sdk](https://github.com/stellar/go/blob/d12c3e18eddfee571b6dc2d1d708936e7701e7c1/protocols/horizon/main.go#L224-L249).

**What version are you on?**
9.1.0

**Additional context**
Was looking for a type to handle the below code, when I realized the fields are missing
```
var horizon = new stellarSdk.Server("https://horizon.stellar.org");
        horizon
          .ledgers()
          .cursor(pagingToken)
          .limit(200)
          .stream({
            onmessage: (ledger: ServerApi.LedgerRecord) => {
              ledger.successful_transaction_count // error TS2339: Property 'successful_transaction_count' does not exist on type 'LedgerRecord'.
...
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -8,7 +8,7 @@ A breaking change will get clearly marked in this log.
 
 ### Fix
 
-- Add missing fields to the `LedgerRecord`: `successful_transaction_count` and `failed_transaction_count`. Note that this also marks several fields as _deprecated_ because they don't actually exist in the Horizon API response:
+- Add missing fields to the `LedgerRecord`: `successful_transaction_count` and `failed_transaction_count` ([#740](https://github.com/stellar/js-stellar-sdk/pull/740)). Note that this also marks several fields as _deprecated_ because they don't actually exist in the Horizon API response:
 
   - `transaction_count`: superceded by the sum of the aforementioned fields
   - `base_fee`: superceded by the `base_fee_in_stroops` field

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -6,6 +6,17 @@ A breaking change will get clearly marked in this log.
 ## Unreleased
 
 
+### Fix
+
+- Add missing fields to the `LedgerRecord`: `successful_transaction_count` and `failed_transaction_count`. Note that this also marks several fields as _deprecated_ because they don't actually exist in the Horizon API response:
+
+  - `transaction_count`: superceded by the sum of the aforementioned fields
+  - `base_fee`: superceded by the `base_fee_in_stroops` field
+  - `base_reserve`: superceded by the `base_reserve_in_stroops` field
+
+These deprecated fields will be removed in the next major version. It's unlikely that this breaking change should affect anyone, as these fields have likely been missing/invalid for some time.
+
+
 ## [v10.0.0](https://github.com/stellar/js-stellar-sdk/compare/v9.1.0...v10.0.0)
 
 This release introduces breaking changes from `stellar-base`. It adds **unconditional support for muxed accounts**. Please refer to the corresponding [release notes](https://github.com/stellar/js-stellar-base/releases/tag/v7.0.0) for details on the breaking changes there.

--- a/src/server_api.ts
+++ b/src/server_api.ts
@@ -148,6 +148,21 @@ export namespace ServerApi {
     header_xdr: string;
     base_fee_in_stroops: number;
     base_reserve_in_stroops: number;
+    /**
+     * @deprecated  This will be removed in the next major version: the property
+     *     no longer exists on the Horizon API
+     */
+    transaction_count: number;
+    /**
+     * @deprecated  This will be removed in the next major version: the property
+     *     no longer exists on the Horizon API
+     */
+    base_fee: number;
+    /**
+     * @deprecated  This will be removed in the next major version: the property
+     *     no longer exists on the Horizon API
+     */
+    base_reserve: string;
 
     effects: CallCollectionFunction<EffectRecord>;
     operations: CallCollectionFunction<OperationRecord>;

--- a/src/server_api.ts
+++ b/src/server_api.ts
@@ -136,14 +136,13 @@ export namespace ServerApi {
     hash: string;
     prev_hash: string;
     sequence: number;
-    transaction_count: number;
+    successful_transaction_count: number;
+    failed_transaction_count: number;
     operation_count: number;
     tx_set_operation_count: number | null;
     closed_at: string;
     total_coins: string;
     fee_pool: string;
-    base_fee: number;
-    base_reserve: string;
     max_tx_set_size: number;
     protocol_version: number;
     header_xdr: string;

